f3/granite.go

```diff
@@ -43,6 +43,33 @@ type GMessage struct {
 	Ticket Ticket
 	// Signature by the sender's public key over Instance || Round || Step || Value.
 	Signature []byte
+	// Evidence is the aggregated list of GossiPBFT msgs that justify this message
+	Evidence AggEvidence
+}
+
+// Aggregated list of GossiPBFT messages with the same instance, round and value. Used as evidence for justification of messages
+type AggEvidence struct {
+	// Enumeration of QUALITY, PREPARE, COMMIT, CONVERGE, DECIDE
+	Step string
+	// Chain of tipsets proposed/voted for finalisation in this instance.
+	// Non-empty: the first entry is the base tipset finalised in instance-1
+	Value ECChain
+	// GossiPBFT instance number
+	Instance uint32
+	// GossiPBFT round
+	Round uint32
+	// Indexes in the base power table of the signers (bitset)
+	Signers BitSet
+	// BLS aggregate signature of signers
+	Signature []byte
+}
+
+func ZeroAggEvidence() AggEvidence {
+	return AggEvidence{}
+}
```

Comment:
I don't think we need this method if the canonical Go zero value is what we want.